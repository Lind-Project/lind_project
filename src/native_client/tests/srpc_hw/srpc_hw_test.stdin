rpc fortytwo * i(0)
rpc helloworld * s(0)
