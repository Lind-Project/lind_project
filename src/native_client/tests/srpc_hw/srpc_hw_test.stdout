rpc call initiated fortytwo::i
rpc call complete fortytwo::i
output 0:  i(42)

rpc call initiated helloworld::s
rpc call complete helloworld::s
output 0:  s("hello, world.")

